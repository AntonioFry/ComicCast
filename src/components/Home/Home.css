.home-section {
  background: url('../../images/Background-stock.jpg');
  max-height: 900px;
  padding: 50px;
}

h2 {
  margin: 0px 0px 10px
}

.suggested-character-heading {
  margin: 100px 0px 0px;
}