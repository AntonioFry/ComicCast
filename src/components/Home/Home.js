import React, { Component } from 'react';
import './Home.css';

class Home extends Component {
  constructor(props) {
    super(props);
    this.state = {
      
    }
  }

  render() {
    const { image, name, biography } = this.props.featuredCharacter;
    console.log(image)
    return (
      <section className="home-section">
        <article className="featured-character-article">
          <img className="featured-character-img" src={image.url} alt="featured character"/>
          <div className="featured-character-info">
            <h2 className="featured-character-text">{name}</h2>
            <p className="featured-character-text">{biography["full-name"]}</p>
          </div>
        </article>
        
      </section>
    )
  }
}

export default Home;