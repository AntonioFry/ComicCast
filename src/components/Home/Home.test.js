import React from 'react'
import { Home } from './Home'
import { shallow } from 'enzyme';

describe('Home', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  })
})