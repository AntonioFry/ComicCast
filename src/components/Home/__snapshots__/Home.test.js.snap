// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match the snapshot 1`] = `
<section
  className="home-section"
>
  <h2
    className="featured-character-heading"
  >
    Featured Character
  </h2>
  <Connect(FeaturedCharacter) />
  <h2
    className="suggested-character-heading"
  >
    Suggested Characters
  </h2>
  <Connect(CharacterCarousel) />
</section>
`;
